apis/tags/v1beta1/tagbinding_identity.go

```diff
@@ -13,3 +13,97 @@
 // limitations under the License.
 
 package v1beta1
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/k8s-config-connector/apis/common"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/apis/common/identity"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	// TagsTagBindingIdentityURL is the format for the externalRef of a TagsTagBinding.
```

Comment:
NIT: Same as TagsTagBinding. I think its useful to comment some metadata about {{tagBinding}} being both service generated and universal. 

Also wonder if it would be nice to leave a comment link to https://docs.cloud.google.com/asset-inventory/docs/asset-names